testers/sql/server/markus_sql_tester.py

```diff
@@ -25,19 +25,19 @@ class MarkusSQLTest(MarkusTest):
     SCHEMA_FILE = 'schema.ddl'
     DATASET_DIR = 'datasets'
 
-    def __init__(self, tester, test_file, data_files, points, test_extra, feedback_open):
-        super().__init__(tester, test_file, data_files, points, test_extra, feedback_open)
+    def __init__(self, test_file, data_files, points, test_extra, feedback_open, **kwargs):
+        super().__init__(test_file, data_files, points, test_extra, feedback_open)
         self.data_file = data_files[0]
-        self.oracle_database = tester.oracle_database
-        self.test_database = tester.test_database
-        self.user_name = tester.user_name
-        self.user_password = tester.user_password
-        self.oracle_connection = tester.oracle_connection
-        self.oracle_cursor = tester.oracle_cursor
-        self.test_connection = tester.test_connection
-        self.test_cursor = tester.test_cursor
-        self.path_to_solution = tester.specs['path_to_solution']
-        self.schema_name = tester.specs['schema_name']
+        self.oracle_database = kwargs['oracle_database']
+        self.test_database = kwargs['test_database']
+        self.user_name = kwargs['user_name']
+        self.user_password = kwargs['user_password']
+        self.oracle_connection = kwargs['oracle_connection']
+        self.oracle_cursor = kwargs['oracle_cursor']
+        self.test_connection = kwargs['test_connection']
+        self.test_cursor = kwargs['test_cursor']
+        self.path_to_solution = kwargs['path_to_solution']
+        self.schema_name = kwargs['schema_name']
```

Comment:
Same issue with using kwargs as above.  I think we need to make all of these explicit keyword arguments.